src/app.ts

```diff
@@ -27,7 +37,7 @@ DBConnection()
   })
   .catch((error: any) => console.log(error));
 
-async function importDataFromFile(filename: string, fn: (data: string) => Promise<void>) {
+async function importDataFromFile(filename: string, fns: ((data: string) => Promise<void>)[]) {
```

Comment:
what is fns?

Reply:
I changed it so we can run multiple functions (fns) on the data from one file. This way, we can create a table that is an amalgamation of groups and interests and add it to TalkingPoints without having to iterate over the entire file again.